README.md

```diff
@@ -110,21 +113,27 @@ Deploy the stack using the following command
 cdk deploy
 ```
 
-### Examples 
+This will provision the following:
 
-To view more examples for how you can leverage `cdk-eks-blueprint`, see the `examples` directory.
+- [x] A new Well-Architected VPC with both Public and Private subnets.
+- [x] A new Well-Architected EKS cluster in the region and account you specify.
+- [x] [ArgoCD](https://argoproj.github.io/argo-cd/) into your cluster to support GitOps deployments. 
+- [x] [Calico](https://docs.projectcalico.org/getting-started/kubernetes/) into your cluster to support Network policies.
+- [x] [Metrics Server](https://github.com/kubernetes-sigs/metrics-server) into your cluster to support metrics collection.
+- [x] AWS and Kubernetes resources needed to forward logs and metrics to [Container Insights](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/deploy-container-insights-EKS.html).
+- [x] AWS and Kubernetes resources needed to support [AWS Load Balancer Controller](https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html).
 
 ---
 
-## Why should I use this framework?  
+## Why should I use this QuickStart?  
 
-SSPs can be difficult to design and build. From an operations perspective, you need to determine the right mix of tools and services you want to include in your platform and how they integrate. You also need to determine how to maintain a fleet of clusters and their associated addons across regions. From a developer perspective, you need to determine how to onboard an operate workloads on the platfrom as well. 
+The ecosystem of tools that have developed around Kubernetes and the Cloud Native Computing Foundation (CNCF) provide cloud engineers with a wealth of choice when it comes to architecting their infrastructure. Determining the right mix of tools and services however, in addition to how they integrate, can be a challenge. As your Kubernetes estate grows, managing configuration for your clusters can also become a challenge. 
 
-The `cdk-eks-blueprint` framework provides logical abstractions and perscriptive guidance for building a well-architected platform, and ultimately, will help accelerate time to market for your platform implementation.
+AWS customers are building internal platforms to tame this complexity, automate the management of their Kubernetes environments, and make it easy for developers to onboard their workloads. These platforms require investment of time and engineering resources to build however. The goal of this QuickStart is to provide customers with a tool chain that can help them deploy a Well-Architected platform on top of EKS with ease. The `cdk-eks-blueprint` framework provides logical abstractions and prescriptive guidance for building a platform. Ultimately, we want to help EKS customers accelerate time to market for their own platform initiatives. 
```

Comment:
minor: "to build however"? May use commas and move "however" to the beginning.

Reply:
fine by me